docs/articles/nunit/getting-started/installation.md

```diff
@@ -1,43 +1,149 @@
+
+## I'm using Visual Studio as my development tool
+
+You can add a NUnit project from the Add New Project dialog. Search for NUnit and choose the NUnit Test Project template.
+
+This will add a new project to your solution, containing a single test class and a single test method. Your project file will have the necessary references to NUnit, the NUnit Test Adapter, the NUnit Analyzer, Microsoft Test SDK and the coverlet code coverage package.
+
+Be aware that the templates behind this may be out of date, so always go to the Manage NuGet Packages dialog and update to the latest version of all packages there.
+
+## I'm using Visual Studio Code as my development tool
+
+Ensure you have the [C# Dev Kit](https://code.visualstudio.com/docs/csharp/get-started) installed. Then Shift-Control-P, type `Net` and select `New Project`. Select `NUnit 3 Test Project` and follow the prompts.
+The results will be exactly the same as if you had used Visual Studio.
+
+Rider has built-in support for NUnit, but are using the same template, so you get the full result.
+
+## I'm using Rider as my development tool
+
+This is the same procedure as for Visual Studio, `Add New Project`, select the `NUnit Test Project` template and follow the prompts.
+The results will be exactly the same as if you had used Visual Studio.
+
+## I prefer to do this using the command line or any of the built-in terminals
+
+From your solution folder, run `dotnet new nunit -o TestProject1` to create a new NUnit test project in the folder `TestProject1`.
+The results will be exactly the same as if you had used Visual Studio.
+
+## Examples of what you get
+
+All of the above will create a new project with the following structure, after updating the packages to the latest versions (of Sept 2023):
+
+```csproj
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>false</IsPackable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
+        <PackageReference Include="NUnit" Version="3.13.3" />
+        <PackageReference Include="NUnit3TestAdapter" Version="4.5.0" />
+        <PackageReference Include="NUnit.Analyzers" Version="3.7.0">
+          <PrivateAssets>all</PrivateAssets>
+          <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+        <PackageReference Include="coverlet.collector" Version="6.0.0">
+          <PrivateAssets>all</PrivateAssets>
+          <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+        </PackageReference>
+     </ItemGroup>
+
+</Project>
+
+```
+
+It will create a `UnitTest1.cs` file with the following contents:
+
+```cs
```

Comment:
❓ Same comment for this - should we maybe place it in the samples?

Reply:
I don't think this should be part of samples, as this is just what is coming out of the template.  Just wanted to show people what it actually is that they will get.  Also makes the page a bit less boring, it adds colors and code, what more could anyone want?